### ISSUE TITLE
Selecting text removes initial selection and cannot copy

### ISSUE THREAD

**Issue by hkdobrev** [Nuclides/github-highlight-selected#11]
I enjoy using the extension, but when I try to just select something to copy it to clipboard I can't. It clears my selection as soon as I release the trackpage/mouse button. Copying while holding down the button works, but it's not ideal.

Is it possible to implement it so the initial native selection remains?

**Comment by xhacker** [comment]
Good point, but I don't think I can reproduce. Can you report your OS, browser, and extension version?

**Comment by hkdobrev** [comment]
I'm afraid I'm using quite a bit of beta software.

Mac OS: 10.11.4 Beta 15E61b
Chrome Version 50.0.2661.18 dev (64-bit)
GitHub Highlight Selected Version 2.1

**Comment by xhacker** [comment]
I still cannot reproduce. Are you encountering the issue on every pages? Are you using any other extensions that may interfere with this?

**Comment by jakeonrails** [comment]
I also just started experiencing this issue.

**Comment by jakeonrails** [comment]
Also, the extension is now affecting comments on PRs and not just the code of the PR. Maybe github changed the HTML structure so the previous selectors are incorrect now?
